src/Refinement/RefinedDiscreteModels.jl

```diff
@@ -0,0 +1,110 @@
+
+
+"""
+Refinement glue between two nested triangulations
+- `f2c_faces_map`     : Given a fine face gid, returns the gid of the coarse face containing it. 
+- `fcell_to_child_id` : Given a fine cell gid, returns the local child id within the coarse cell containing it.
+- `f2c_ref_cell_map`  : Ref coordinate map between the fcells and their parent ccell, i.e 
+                          for each fine cell defines Φ st. x_c = Φ(x_f)
+"""
+struct RefinementGlue{A,B,C} <: GridapType
+  f2c_faces_map::A
+  fcell_to_child_id::B
+  f2c_ref_cell_map::C
+
+  function RefinementGlue(f2c_faces_map,
+                          fcell_to_child_id,
+                          f2c_ref_cell_map)
+    A = typeof(f2c_faces_map)
+    B = typeof(fcell_to_child_id)
+    C = typeof(f2c_ref_cell_map)
+    new{A,B,C}(f2c_faces_map,fcell_to_child_id,f2c_ref_cell_map)
+  end
+end
+
+"""
+Ref coordinate map from fcell ref coords to ccell ref coords.
+Size -> Number of children per coarse cell.
+"""
+function get_f2c_ref_cell_map(reffe,ref)
```

Comment:
Perhaps we may type annotate the function parameters. 

Rename `ref` by `nrefs`?

Reply:
To-think: a better name for `get_f2c_ref_cell_map`

What about `get_refinement_rule_reference_cell_map`?

On another note. I think that here we are handling two different concepts: "reference" and "refinement". We have to be careful with the notation. Example: `ref` means refinement or reference?

Reply:
Yes, the names of these functions need to be thought out. I'll think about it. This is just a first version.